src/exercises/Exercises.js

```diff
@@ -49,6 +81,16 @@ export default function Exercises({
   const [isCorrect, setIsCorrect] = useState(false);
   const [showFeedbackButtons, setShowFeedbackButtons] = useState(false);
   const [reload, setReload] = useState(false);
+  const [audioQuestionnaireMessageOpen, setAudioQuestionnaireMessageOpen] = useState(false);
```

Comment:
this and the next don't change state ever in this component. why are they state? you can make them as state if you need them in `QuestionnaireMessage`.